src/app/campaign-details/campaign-details.component.html

```diff
@@ -34,16 +34,26 @@
       >
       </biggive-branded-image>
 
+      <biggive-heading
+        space-above=0
+        space-below=0
+        colour="primary"
+        html-element="h1"
+        size="3"
+        align="left"
+        [text]="campaign.title"
+      >
+      </biggive-heading>
+
       <biggive-formatted-text
         default-text-colour="black">
-        <h1>{{ campaign.title }}</h1>
```

Comment:
this looks fine, not sure which part of the Jira task it relates to though.

Reply:
Relates to point 3. @bdsl